CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ We welcome contributions of:
  * **Documentation Improvements**: You can submit improvements to either:
      * Ozone website. Instructions are here: [Modifying the Ozone Website](https://cwiki.apache.org/confluence/display/HADOOP/Modifying+the+Ozone+Website)
```

Comment:
```suggestion
     * Ozone website. Instructions are here: [Modifying the Ozone Website](https://cwiki.apache.org/confluence/display/OZONE/Modifying+the+Ozone+Website)
```
This link also needs to be updated `HADOOP` -> `OZONE`

Reply:
Sure.